arelle/ValidateFilingText.py

```diff
@@ -693,7 +695,7 @@ def validateFootnote(modelXbrl, footnote):
                 _("%(validatedObjectLabel)s causes the XML error %(error)s"),
                 modelObject=footnote, validatedObjectLabel=validatedObjectLabel,
                 error=', '.join(e.message for e in edbodyDTD.error_log.filter_from_errors()))
-        validateHtmlContent(modelXbrl, footnote, footnoteHtml, validatedObjectLabel, modelXbrl.modelManager.disclosureSystem.validationType + ".6.05.34.")
+        validateHtmlContent(modelXbrl, footnote, footnoteHtml, validatedObjectLabel, modelXbrl.modelManager.disclosureSystem.validationType + ".6.05.34.", supportedImageTypes=supportedImageTypes)
```

Comment:
This line causes an error because `supportedImageTypes` is not defined at this point in the code.